roles/frameworks/templates/cassandra-config.j2

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
even expose the minimum required configuration values would be nice with the default in the config.json e.g.
"framework","cluster-name","zk-timeout-ms","node-count","seed-count","health-check-interval-seconds","bootstrap-grace-time-seconds","data-directory","resources"